pkg/testing/fixture.go

```diff
@@ -1196,6 +1196,31 @@ func performConfigure(ctx context.Context, c client.Client, cfg string, timeout
 	return nil
 }
 
+// createTempDir creates a temporary directory that will be
+// removed after the tests passes. If the test fails, the
+// directory is kept for further investigation.
+//
+// If the test is run with -v and fails the temporary directory is logged
```

Comment:
Since you stated that the test must be run with the -v flag you might want to check for [`t.Verbose()`](https://pkg.go.dev/testing#Verbose) explicitly. I don't mind just rephrasing the comment to something like `If the test fails the temporary directory is logged. Use the -v flag to check the directory name.`

Reply:
Because it uses `t.Logf` it is only shown when the `-v` flag is passed, so checking `t.Verbose()` seems redundant here.

Is there a corner case I'm not seeing?

I like the change in the copy, I'll implement it. Thanks!

Reply:
That is exactly the point I was trying to get to, I think just rephrasing the comment is enough, I also don't see a need to check for t.Verbose explicitly.